components/webgpu/lib.rs

```diff
@@ -566,6 +567,17 @@ impl<'a> WGPU<'a> {
                             },
                             None => None,
                         };
+
+                        let vert_buffers = vertex_state
+                            .1
+                            .iter()
+                            .map(|buffer| wgpu_core::pipeline::VertexBufferLayoutDescriptor {
```

Comment:
it's a good habit to destructurize tuples when pattern mapping, i.e. it could be `|(array_stride, step_mode, attributes)|` here